admin/app/views/workarea/admin/tax_rates/edit.html.haml

```diff
@@ -21,7 +21,7 @@
         .grid__cell.grid__cell--50
           .property
             = label_tag 'rate_country_percentage', t('workarea.admin.tax_rates.index.table.country_percentage'), class: 'property__name'
-            = number_field_tag 'rate[country_percentage]', @rate.country_percentage * 100, min: 0, max: 100, step: 0.01, class: 'text-box text-box--i18n'
+            = number_field_tag 'rate[country_percentage]', @rate.country_percentage * 100, min: 0, max: 100, step: 'any', class: 'text-box text-box--i18n'
```

Comment:
The default for these fields is `1`, so making this "any" allows any kind of numerical input. This should resolve any current and future issues surrounding editing taxes in admin...